proto/poktroll/application/types.proto

```diff
@@ -22,6 +22,7 @@ message Application {
   // Total amount of staked uPOKT
   cosmos.base.v1beta1.Coin stake = 2;
 
+  // TODO_MAINNET: Refactor this to be a single service config field.
```

Comment:
#PUC why this is necessary?

Reply:
Perhaps I misunderstood a part of your previous feedback.

Reply:
```suggestion
```